client/apps/edit/components/admin/components/verticals_tags.jsx

```diff
@@ -0,0 +1,112 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import { Col, Row } from 'react-styled-flexboxgrid'
+import { data as sd } from 'sharify'
+import Verticals from '../../../../../collections/verticals.coffee'
+import { AutocompleteInlineList } from '/client/components/autocomplete2/inline_list'
+
+export class AdminVerticalsTags extends Component {
+  static propTypes = {
+    article: PropTypes.object,
+    onChange: PropTypes.func
+  }
+
+  constructor (props) {
+    super(props)
+    const { article } = props
+
+    this.state = {
+      vertical: article.get('vertical') || null,
+      verticals: []
+    }
+  }
```

Comment:
I think this constructor is unnecessary, right? I'm not so picky about it, up to you.

Reply:
good catch